src/core/filesystem.cc

```diff
@@ -991,7 +991,15 @@ GetDirectoryFiles(const std::string& path, std::set<std::string>* files)
 {
   FileSystem* fs;
   RETURN_IF_ERROR(GetFileSystem(path, &fs));
-  return fs->GetDirectoryFiles(path, files);
+  std::set<std::string> all_files;
```

Comment:
Don't do this unconditionally. Add a "skip_hidden_files = false" parameter to the function.

Reply:
Done